pages/linux/quotacheck.md

```diff
@@ -0,0 +1,28 @@
+# quotacheck
+
+> Scan a filesystem for disk usage; create, check and repair quota files.
+> Should only be run by the super-user.
+
+- Check all mounted non-NFS filesystems:
```

Comment:
I think that maybe it should be noted that actual usage may change while running quotacheck on an active file system. 
How would I best convey this, if you agree it needs to be added, @waldyrious ?